alpha/engagements/2024/Rust Foundation/update-2024-09.md

```diff
@@ -0,0 +1,35 @@
+A successful RustConf 2024 consumed a bunch of the team's time this month, but we definitely got some wins this month. One win in particular was an agreement to use TUF for crate signing and mirroring.
+
+## Engineering
+
+### Rust TUF
+
+A breakthrough occurred at [RustConf 2024](https://rustconf.com) - an agreement between members of the cargo team, crates team and the Foundation's security initiative agreed to implement [The Update Framework](https://theupdateframework.io/) (TUF) for Rust in order to move forward with crate signing and mirroring.  This would replace the original proposal outlined in the [PKI RFC](https://github.com/rust-lang/rfcs/pull/3579). This agreement amongst the affected parties is actually a big milestone for progress.
+
+There are some technical hurdles to overcome to make this goal of using TUF achievable. One of these hurdles is ensuring that the size of the crate payload metadata is not increased at a noticeable rate. Right now it is estimated that as-is, 1MB of data would be appended to every crate download, which we want to try to reduce, if possible. In order to reduce the payload size required to successfully use TUF for Rust, we want to implement TAP16. TAP16 “proposes a method for reducing the size of snapshot metadata a client must download without significantly weakening the security properties of TUF”, by using merkle trees.
+
+### Publish Notification Emails
+
+Tobias has [pushed](https://github.com/rust-lang/crates.io/pull/9341/) his implementation of notifying crate owners when their crates have been published, including, to avoid potential notification overload backlash, a mechanism to[opt-out](https://github.com/rust-lang/crates.io/pull/9359), if desired. These notification emails will allow crate owners to ensure that no one is publishing their crates without their knowledge.
```

Comment:
Missing space

```suggestion
Tobias has [pushed](https://github.com/rust-lang/crates.io/pull/9341/) his implementation of notifying crate owners when their crates have been published, including, to avoid potential notification overload backlash, a mechanism to [opt-out](https://github.com/rust-lang/crates.io/pull/9359), if desired. These notification emails will allow crate owners to ensure that no one is publishing their crates without their knowledge.
```